package.json

```diff
@@ -2145,7 +2145,6 @@
         "react-svg-pan-zoom": "3.9.0",
         "react-svgmt": "1.1.11",
         "react-virtualized": "^9.21.1",
-        "react17": "npm:react@^17.0.2",
```

Comment:
I think that this was added due to the peerDependency issues that I believe that I fixed with my previous package changes. So no need to have a useless react17 added anymore.